index.html

```diff
@@ -27,15 +27,16 @@
     <title>Francisco's Portfolio</title>
 </head>
 <body>
-        <header class="nav-bar">
+        <header id="nav" class="nav-bar">
             <h3>Francisco Ponce</h3>
+            <img id="close" src="./assets/closing-btn.svg" alt="Close menu" class="closing-btn">
             <ul class="desktop-only">
                 <li><a class="link" href="#portfolio">Portfolio</a></li>
                 <li><a class="link" href="#about-me">About</a></li>
                 <li><a class="link" href="#contact">Contact</a></li>
                 <li><a class="link" href="#contact"><i class="far fa-envelope fa-lg"></i></a></li>
```

Comment:
- [ ] Your design is well, Please notice though that you have an extra `li` tag in the hamburger menu; this is causing your menu to leave an extra border below the "Contact". You have set the `display` to `none` for the envelope which is good although you also need to handle the `border-bottom` on the `li` element itself. Please kindly fix this issue as this doesn't match the reference design.

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/150873267-a4fe623a-e26f-46f8-977e-60696e769ebb.png)|![image](https://user-images.githubusercontent.com/18730106/150873317-bb626c7f-51b1-4fd9-b0fe-eaa43e8bcc1b.png)